package.json

```diff
@@ -171,6 +171,7 @@
   },
   "resolutions": {
     "@types/node": "~14.14.45",
+    "ansi-escapes/type-fest": "^2.0.0",
```

Comment:
I just did the same and it works ;D

There is also `jasmine` related error. I think it can be ignored with `@ts-expect-error`.